virttest/virt_vm.py

```diff
@@ -638,6 +638,8 @@ def get_address(self, index=0):
         :raise VMAddressVerificationError: If the MAC-IP address mapping cannot
                 be verified (using arping)
         """
+
+        flexible_index = self.params.get("flexible_nic_index", 'no')
```

Comment:
@will-Do Have come up with the Version 2(https://github.com/avocado-framework/avocado-vt/issues/816) with all the changes implemented, hence closing this PR